code/src/App.css

```diff
@@ -0,0 +1,108 @@
+header {
+  display: flex;
+}
+
+.header-title {
+  font-size: 29px;
+  margin: auto;
+  padding-top: 20px;
+  color: #ECCCCF;
+  opacity: 80%;
+}
+
+.page-container {
+  margin: 15px 30px 15px 30px;
+  padding: 25px;
+  border-radius: 20px;
+  /* From https://css.glass */
+  background: rgba(255, 255, 255, 0.45);
+  border-radius: 16px;
+  box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+  backdrop-filter: blur(6.3px);
+  -webkit-backdrop-filter: blur(6.3px);
+  border: 1px solid rgba(255, 255, 255, 0.12);
```

Comment:
Im not sure how to make comments on several commits in a file but this is general about the styling.. FIrst of all i really love your bg-image!! It really talks with the theme of the survey and it gets very dreamy <3 The contrasts to the text elements could be a little bigger, its hard to read the text especially the footer content. You could easily fix this by make the backdrop-filter: blur into maybe 80px instead then its much more readable.